.gitignore

```diff
@@ -0,0 +1,4 @@
+/.config
+/fs0/nginx/logs/
```

Comment:
Copy-paste? For shame!

I would do a separate commit for ignoring output files.

Reply:
It should be `/fs0/`.